src/helper-methods.js

```diff
@@ -0,0 +1,67 @@
+import merge from "lodash/object/merge";
+import { Chart, Style } from "victory-util";
+import { Transform } from "react-art";
+
+
+module.exports = {
+  getStyles(props, defaultStyles) {
+    const style = props.style ? merge({}, defaultStyles, props.style) : defaultStyles;
+    return Style.removeInvisible(Chart.evaluateStyle(style));
+  },
+
+  getContentLength(props) {
+    if (props.children) {
```

Comment:
Also, if all you need is `props.children` why not pass that in instead of the whole props object?